src/Airship/Internal/Decision.hs

```diff
@@ -696,7 +693,16 @@ o16 r = do
     req <- lift request
     if requestMethod req == HTTP.methodPut
         then o14 r
-        else o18 r
+        else o17 r
+
+o17 r@Resource{..} = do
+    trace "o17"
```

Comment:
@tmcgilchrist Apparently webmachine is implemented based strictly on RFC 2616, which does not discuss HTTP PATCH